sesam.py

```diff
@@ -23,6 +23,7 @@
 from fnmatch import fnmatch
 from decimal import Decimal
 import pprint
+from jsonformat import format_object
 
 sesam_version = "1.15.42"
```

Comment:
1.16.1 (im guessing this is related to the changes in the frontend so this is a bugfix?)

Reply:
It's technically a fix for a change we did on the backend, we sort dictionary keys differently but this has caused changes when doing "status" (reordering of keys in embedded entities, if one uses properties with underscores in the embedded entities)

Reply:
I've now bumped the version to 1.16.1